contrib/epee/src/net_ssl.cpp

```diff
@@ -254,6 +254,10 @@ boost::asio::ssl::context ssl_options_t::create_context() const
 
 void ssl_authentication_t::use_ssl_certificate(boost::asio::ssl::context &ssl_context) const
 {
+  if (!private_key_passphrase.empty())
+    ssl_context.set_password_callback([this](size_t max_length, boost::asio::ssl::context::password_purpose purpose)->std::string{
```

Comment:
Is the `wipeable_string` instance really that useful if a copy is being made anyway?